plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_group_members
+short_description: Manage group members on GitLab Server
+description:
+    - This module allows to add and remove members to/from a group, or change a member's access level in a group on GitLab.
+version_added: '1.2.0'
+author: Zainab Alsaffar (@zanssa)
+requirements:
+    - python-gitlab python module <= 1.15.0
+    - administrator rights on the GitLab server
+options:
+    server_url:
+        description:
+            - URL of a GitLab server.
+        required: true
+        type: str
+    access_token:
+        description:
+            - A personal access token to authenticate with the GitLab API.
+        required: true
+        type: str
+    gitlab_group:
+        description:
+            - The name of the GitLab group the member is added to/removed from.
+        required: true
+        type: str
+    gitlab_user:
+        description:
+            - The username of the member to add to/remove from the GitLab group.
+        required: true
+        type: str
+    access_level:
+        description:
+            - The access level for the user.
+            - Required if I(state=present), user state is set to present.
+        type: str
+        choices: ['guest', 'reporter', 'developer', 'maintainer', 'owner']
+    state:
+        description:
+            - State of the member in the group.
+            - On C(present), it adds a user to a GitLab group.
+            - On C(absent), it removes a user from a GitLab group.
+        choices: ['present', 'absent']
+        default: 'present'
+        type: str
+notes:
+    - Supports C(check_mode).
+'''
+
+EXAMPLES = '''
+- name: Add a user to a GitLab Group
+  community.general.gitlab_group_members:
+    server_url: 'https://gitlab.example.com'
+    access_token: 'Your-Private-Token'
+    gitlab_group: groupname
+    gitlab_user: username
+    access_level: developer
+    state: present
+
+- name: Remove a user from a GitLab Group
+  community.general.gitlab_group_members:
+    server_url: 'https://gitlab.example.com'
+    access_token: 'Your-Private-Token'
+    gitlab_group: groupname
+    gitlab_user: username
+    state: absent
+'''
+
+RETURN = '''
+'''
```

Comment:
```suggestion
RETURN = r''' # '''
```
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block